src/WireMock.Net/Matchers/XPathMatcher.cs

```diff
@@ -19,16 +20,22 @@ namespace WireMock.Matchers;
 /// <seealso cref="IStringMatcher" />
 public class XPathMatcher : IStringMatcher
 {
+    private readonly List<XmlNamespace> _xmlNamespaceMap;

```

Comment:
maybe better to define this as nullable?

Reply:
And must this be a list? Or is array also fine?

Reply:
I can change this if you want. To me it is an implementation detail and it prevents NullReferenceExceptions.